about/release_policy.rst

```diff
@@ -77,41 +77,44 @@ If you experience an issue using an older patch release, please upgrade to the
-| Godot 1.0   | December 2014        | |eol| No longer supported.                                               |
-+-------------+----------------------+--------------------------------------------------------------------------+
++--------------+----------------------+--------------------------------------------------------------------------+
+| **Version**  | **Release date**     | **Support level**                                                        |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 4.2    | November 2023        | |unstable| *Development.* Receives new features, usability and           |
+| (`master`)   | (estimate)           | performance improvements, as well as bug fixes, while under development. |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 4.1    | July 2023            | |supported| Receives fixes for bugs and security issues, as well as      |
+|              |                      | patches that enable platform support.                                    |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 4.0    | March 2023           | |supported| Receives fixes for bugs and security issues, as well as      |
+|              |                      | patches that enable platform support.                                    |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 3.6    | Q3 2023 (estimate)   | |supported| *Beta.* Receives new features, usability and performance     |
+| (`3.x`, LTS) |                      | improvements, as well as bug fixes, while under development.             |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 3.5    | August 2022          | |supported| Receives fixes for bugs and security issues, as well as      |
+|              |                      | patches that enable platform support.                                    |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 3.4    | November 2021        | |eol| No longer supported, as fully superseded by the compatible 3.5     |
+|              |                      | release (last update: 3.4.5).                                            |
++--------------+----------------------+--------------------------------------------------------------------------+
+| Godot 3.3    | April 2021           | |eol| No longer supported, as fully superseded by the compatible 3.4     |
+|              |                      | release (last update: 3.3.4).                                            |
```

Comment:
I wonder if we want to simplify those by now? Not sure why we wrote it this way compared to previous 3.x releases.
```suggestion
| Godot 3.4    | November 2021        | |eol| No longer supported (last update: 3.4.5).                          |
+--------------+----------------------+--------------------------------------------------------------------------+
| Godot 3.3    | April 2021           | |eol| No longer supported (last update: 3.3.4).                          |
```

Reply:
I think we may want to move deprecated versions to a separate table that is a bit more compact. With the upcoming pace of 3 minor releases of 4.x per year this table will grow exponentially.

Reply:
Yeah it makes sense to have the upcoming and supported releases in one table, and the EOL ones in another.